src/platforms/android/guides/okhttp/config.yml

```diff
@@ -0,0 +1,6 @@
+title: OkHttp
```

Comment:
@PeloWriter I'm not sure this should be a Guide, it's a different package that users should install along with the standard Android SDK, it does not work standalone, it still requires the Android SDK, how do you suggest that?

Reply:
Is it similar to the Java logging [integrations?](https://docs.sentry.io/platforms/java/guides/log4j2/) We maintain those as guides.

Reply:
not really, a Java logging integration works standalone, this is just a 3rd party library (HTTP Client library) with our support for auto instrumenting Spans for Performance and adds Breadcrumbs for HTTP requests, this is also a client-side library different than a server-side HTTP library

Reply:
In that case, does it make better sense in Configuration > Integrations?

Reply:
it'd but I'd suggest keeping it like this right now after some thought, the reason is, ideally, we'd also capture exceptions when there's an HTTP error in the future, this integration does not do it right now, but likely to be expanded, unless folks really prefer Integrations, @bruno-garcia

Reply:
For me, the [guide page](https://sentry-docs-git-feat-okhttp.sentry.dev/platforms/android/guides/okhttp/) looks a bit weird. I'm not sure if a guide is the right place to put this. There are a few things misleading to me:

1. The configuration code sample makes sense on its own, but with the sentence above `Configuration should happen as early as possible in your application's lifecycle.` it's a bit confusing to me because it doesn't mention the overall SDK configuration.
2. Verify: The description and the sample code don't really match. The description states detailed information, but I wonder where exactly can I find this detailed info? 
3. Next steps: They don't really make sense to me in combination with the OkHttp details.

Maybe it's better to add this as an integration?

Reply:
don't disagree, but it's not an Integration (if we consider the Integration interface from the Unified API) either, but an Interceptor that they should add to their own Http client, so yeah it'd be a bit hidden as well under Integrations as well I guess, really taking suggestions here too.

Reply:
I think if it's going to grow in functionality, we keep it as a guide, but we'll need to be explicit to manage the issues @philipphofmann raises as well as content we should add for clarity. My suggestions:
1. add this text to `include/getting-started-primer/` - "The OkHttp integration is a third party library that supports for instrumenting spans to enable performance monitoring. In addition, the integration adds breadcrumbs for HTTP requests."
2. add this text to the include to `include/getting-started-configuration/` -
"<Note>

The OkHttp integration is a package you can install along with our Sentry Android SDK. It does currently not work as a standalone, it still requires the [Android SDK](/platforms/android/).

</Note>"
3. Verification - would this error not display in sentry.io? Because the content follows directs users there.

4. Next steps defaults to Android code snippets, so isn't incorrect in terms of configuration given that we've laid the groundwork to understand the relationship between the Android SDK and the integration. However, we can turn those pages off if that's better.

Reply:
so what I've done is pretty much duplicating the common config text to the OkHttp Guide but fixing the wording for each section, Install, Config and Verify, so they do make sense when people are navigating thru the OkHttp guide, I guess it's better now :) maybe some typos that I'd get reviewed by u all, but the structure now does make sense for me.